dist/leaflet.css

```diff
@@ -152,7 +152,7 @@
 .leaflet-fade-anim .leaflet-map-pane .leaflet-popup {
 	opacity: 1;
 	}
-.leaflet-zoom-animated {
+.leaflet-zoom-anim .leaflet-zoom-animated {

```

Comment:
leaflet-zoom-anim class is only added during animation, would this kinda defeat the purpose (according to that article that says you don't want to put the property immediately before running the effect)?